CMakeLists.txt

```diff
@@ -530,6 +530,7 @@ if(ENABLE_CPACK)
           "python-qtpy,"
           "python-scipy,"
           "python-pycifrw (>= 4.2.1),"
+          "python-toml,"
```

Comment:
Can we add `python3-toml` to the list on line 510 too?

Reply:
I can't seem to find a `python3-toml` package, additional `python-toml` has a dependency on `python2-toml` so it looks like it pulls in the appropriate version for RHEL?

Reply:
I think these are the Debian dependencies not RHEL. Don't worry about Python 3 RHEL here.

Reply:
Oh, helps if I look at the filename before I comment haha 🤦‍♂ 

I'll also add it to the deb deps too